File: Jewels and Stones.cpp
class Solution {
public:
    int numJewelsInStones(string J, string S) {
        bitset<128> j;
        int x;
        int count=0;
        for(int i=0;i<J.length();i++)
        {
            x=J[i];
            j[x]=1;
        }
        for(int i=0;i<S.length();i++)
        {
            x=S[i];
            if(j[x]==1)
                count++;
        }
        return count;
        
    }
};

File: Golang Solutions/Jewels and Stones.go
func numJewelsInStones(J string, S string) int {
    
    var j[128] bool
    var x int 
    var count int =0
    for i:=0;i<len(J);i++{
        x=int(J[i])
        j[x]=true
    }
    for i:=0;i<len(S);i++{
        x=int(S[i])
        if j[x]==true{
            count++
        }
            
    }
    return count
    
}